internal/store/deployment.go

```diff
@@ -122,6 +122,35 @@ var (
 				}
 			}),
 		},
+		// This all-in-one metric family contains all conditions for extensibility.
+		// Third party plugin may report customized condition for deployments,
+		// and Kubernetes may add new apps conditions in future.
+		{
+			Name: "kube_deployment_status_condition",
+			Type: metric.Gauge,
+			Help: "The condition of a deployment.",
+			GenerateFunc: wrapDeploymentFunc(func(d *v1.Deployment) *metric.Family {
+				ms := make([]*metric.Metric, len(d.Status.Conditions)*len(conditionStatuses))
+
+				// Collect node conditions and while default to false.
```

Comment:
Hah, did I just spot a copypasta (Man, am I guilty of this!) here :) ? Let's have a more descriptive and accurate comment OR not have a comment at all. Your call.

(I also agree that the comment in `internal/store/node.go` needs to be better).

Reply:
what, no, i wouldn't copy and paste and forget to update a comment! :sob: 

hpa has no comments, and I don't think this one is accurate anyways. I'll remove